### PR TITLE
[infra] Additional option for ARM32 makefile

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise Makefile.arm32 to accept additional
NNCC_CFG_OPTION_EXTRA option from caller.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Review by hyunsik-yoon — APPROVED** [review]

File: infra/nncc/Makefile.arm32 [hunk 1]
```diff
@@ -9,6 +9,7 @@ CURRENT_DIR=$(shell pwd)
 BUILDFOLDER=build
 ARM32_FOLDER=arm32
 ROOTFS_ARM?=$(CURRENT_DIR)/tools/cross/rootfs/arm
+NNCC_CFG_OPTION_ADD?=
```
Comment (chunseoklee) [inline infra/nncc/Makefile.arm32:12]: I am not sure the usage of the varible, but how about name NNCC_CFG_OPTION_EXTRA or something if this is for a sort of extra option ?

Comment (seanshpark) [reply infra/nncc/Makefile.arm32:12]: > I am not sure the usage of the varible,

This is to provide `additional` option from the user(caller) of this Makefile.arm32.

like
```
NNCC_CFG_OPTION_ADD=-DCMAKE_DO_THIS=THAT make -f infra/nncc/Makefile.arm32 all
```

`NNCC_CFG_OPTION_EXTRA` also looks OK. If @hyunsik-yoon also agrees I'll change it.

Comment (chunseoklee) [reply infra/nncc/Makefile.arm32:12]: @hyunsik-yoon PTAL

Comment (hyunsik-yoon) [reply infra/nncc/Makefile.arm32:12]: `NNCC_CFG_OPTION_EXTRA` seems better to me. I thought that `ADD` meant adding some more cfg options from users. 😹

Comment (seanshpark) [reply infra/nncc/Makefile.arm32:12]: > adding some more cfg options from users

Yes, it's adding more cfg options... anyway I'll revise this :)

**Review by chunseoklee — APPROVED** [review]

**Review by hyunsik-yoon — APPROVED** [review]
LGTM

**Pull request merged**